### PR TITLE
BZ1867690: adding default admission plug-ins

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1867690

4.6+

Preview: https://deploy-preview-34850--osdocs.netlify.app/openshift-enterprise/latest/architecture/admission-plug-ins?utm_source=github&utm_campaign=bot_dp#admission-plug-ins-default_admission-plug-ins

### CODE REVIEW

**Comment by xingxingxia** [comment]
> gangwgr commented 2 hours ago
> @kalexand-rh: Need PR review. Please update

@gangwgr , as synced in Slack channel: if doc PR is correct AND complete AND precise, add lgtm in doc PR, then move doc bug to verified. Otherwise, please comment in doc PR to let the doc PR author update, and keep doc bug not verified until the PR is updated that can finally look good to you.

**Comment by gangwgr** [comment]
Find the below admission  plugins details.
$ oc get clusterversion
NAME      VERSION                             AVAILABLE   PROGRESSING   SINCE   STATUS
version   4.6.0-0.nightly-2021-08-09-092758   True        False         20m     Cluster version is 4.6.0-0.nightly-2021-08-09-092758

$ oc logs -n openshift-kube-apiserver $(oc get pod -n openshift-kube-apiserver -l apiserver | sed '1d' | awk '{print $1}' | head -1) -c kube-apiserver | grep -E -m2 'mutating admission controller|validating admission controller'
I0810 06:39:58.152858      18 plugins.go:158] Loaded 19 mutating admission controller(s) successfully in the following order: NamespaceLifecycle,LimitRanger,ServiceAccount,NodeRestriction,TaintNodesByCondition,PodNodeSelector,Priority,DefaultTolerationSeconds,PodTolerationRestriction,PersistentVolumeLabel,DefaultStorageClass,StorageObjectInUseProtection,RuntimeClass,DefaultIngressClass,scheduling.openshift.io/OriginPodNodeEnvironment,image.openshift.io/ImagePolicy,security.openshift.io/SecurityContextConstraint,security.openshift.io/DefaultSecurityContextConstraints,MutatingAdmissionWebhook.
I0810 06:39:58.152882      18 plugins.go:161] Loaded 35 validating admission controller(s) successfully in the following order: LimitRanger,ServiceAccount,PodNodeSelector,Priority,PodTolerationRestriction,OwnerReferencesPermissionEnforcement,PersistentVolumeClaimResize,RuntimeClass,CertificateApproval,CertificateSigning,CertificateSubjectRestriction,authorization.openshift.io/RestrictSubjectBindings,scheduling.openshift.io/OriginPodNodeEnvironment,network.openshift.io/ExternalIPRanger,network.openshift.io/RestrictedEndpointsAdmission,image.openshift.io/ImagePolicy,security.openshift.io/SecurityContextConstraint,security.openshift.io/SCCExecRestrictions,route.openshift.io/IngressAdmission,config.openshift.io/ValidateAPIServer,config.openshift.io/ValidateAuthentication,config.openshift.io/ValidateFeatureGate,config.openshift.io/ValidateConsole,config.openshift.io/ValidateImage,config.openshift.io/ValidateOAuth,config.openshift.io/ValidateProject,config.openshift.io/DenyDeleteClusterConfiguration,config.openshift.io/ValidateScheduler,quota.openshift.io/ValidateClusterResourceQuota,security.openshift.io/ValidateSecurityContextConstraints,authorization.openshift.io/ValidateRoleBindingRestriction,config.openshift.io/ValidateNetwork,ValidatingAdmissionWebhook,ResourceQuota,quota.openshift.io/ClusterResourceQuota.

File: modules/admission-plug-ins-default.adoc [hunk 1]
```diff
@@ -6,4 +6,73 @@
 = Default admission plug-ins
 
 //Future xref - A set of default admission plug-ins is enabled in {product-title} {product-version}. These default plug-ins contribute to fundamental control plane functionality, such as ingress policy, xref:../nodes/clusters/nodes-cluster-overcommit.adoc#nodes-cluster-resource-override_nodes-cluster-overcommit[cluster resource limit override] and quota policy.
-A set of default admission plug-ins is enabled in {product-title} {product-version}. These default plug-ins contribute to fundamental control plane functionality, such as ingress policy, cluster resource limit override and quota policy.
+Default validating and admission plug-ins are enabled in {product-title} {product-version}. These default plug-ins contribute to fundamental control plane functionality, such as ingress policy, cluster resource limit override and quota policy. The following lists contain the default admission plug-ins:
```
Comment (mburke5678) [inline modules/admission-plug-ins-default.adoc:9]: Something about _The following lists contain _ doesn't sound right.  _The following are the default admission lug-ins:_??

**Comment by mburke5678** [comment]
@kalexand-rh One nit. Otherwise LGTM

File: modules/admission-plug-ins-default.adoc (hunk 1 above)
Comment (kalexand-rh) [reply modules/admission-plug-ins-default.adoc:9]: Every instance of "following" in the "Lead-in wording" section of the ISG is followed by a noun.

**Pull request merged**